TimeLog.ReportingApi.SDK/Project.cs

```diff
@@ -262,6 +262,7 @@ public Project(XmlNode node, XmlNamespaceManager namespaceManager)
         /// <summary>
         /// Gets or sets the related project number
         /// </summary>
+        [XmlNodeName("No")]
         public string ProjectNo { get; set; }

```

Comment:
Why not just change the property name instead?

Reply:
Do we need to consider backwards compatibility? If someone has already built an app using ProjectNo, they would have to resolve the breaking changes in the next update.

Reply:
I agree. However, I propose we consider backwards compatibility in the API over the SDK. Unfortunately, this bug has been there for a while I guess. I rather want the SDK to be one-to-one with the API. What's your take?

Reply:
I agree, I believe standardizing the names in SDK and API is preferable. It is a straight forward rule that is easy to apply. 
I have reverted the previous code change and updated the pull request, please review again, thanks!